docs/src/examples/QasSelect/Searchable.vue

```diff
@@ -1,14 +1,19 @@
 <template>
   <div class="container q-py-lg">
     <div>
-      <qas-select v-model="model" label="Meu select!" :options="options" use-search />
+      <qas-select v-model="model" label="Meu select" :options="options" use-search />
       Model: {{ model }}
     </div>
 
     <div class="q-mt-lg">
       <qas-select v-model="model2" label="Meu select multiple" multiple :options="options" use-search />
       Model múltiplo: {{ model2 }}
     </div>
+
+    <div class="q-mt-lg">
+      <qas-select v-model="model3" label="Meu select com busca habilitado automaticamente a partir de 10 options" :options="options2" use-search />
```

Comment:
setando o `use-search` ele não vai ser sempre de busca? caso ele adicione esse comportamento, que acima de 10 adiciona o search, acredito q ele deveria ser true como default para padronização

Reply:
Não, no exemplo coloquei errado, não precisaria esse use-search neste caso, mas o controle interno, n tem a ver com a prop

Reply:
mas essa prop é usada pra habilitar a busca, deveria ter haver não?

Reply:
**É que a regra é:** a partir de 10 opções habilita a busca. Se o `use-search` for sempre `true` como padrão, não há porque ter essa regra.

Em conversas com os designers, para select's com menos de 10 opções meio que torna a busca desnecessária. Por isso não é por default `true`.